configs/bsc/wsteth-mainnet.yml

```diff
@@ -0,0 +1,666 @@
+    - &l2wstETHTokenImpl "0x451d447776778870bdfe76d031689703aba73ee5"
+    - &l2NttManager "0x6981F5621691CBfE3DdD524dE71076b79F0A0278"
+    - &l2NttManagerImpl "0xe82c2a5846cfb6d8683d6b636719e7aa61486838"
+    - &l2WormholeTransceiver "0xbe3F7e06872E0dF6CD7FF35B7aa4Bb1446DC9986"
+    - &l2WormholeTransceiverImpl "0xb948a93827d68a82F6513Ad178964Da487fe2BD9"
+    - &l2AxelarTransceiver "0x723AEAD29acee7E9281C32D11eA4ed0070c41B13"
+    - &l2AxelarTransceiverImpl "0xa1ebb6a4b856df8bf6c3aca88a9115a9ab3b2e02"
+    # Stateless libraries
+    # &l2TransceiverStructs "0xf0396a8077eda579f657B5E6F3c3F5e8EE81972b"
+    # &l2TransceiverStructsAxelar "0x27a3daf3b243104e9b0afae6b56026a416b852c9"
+
+l1:
+  rpcUrl: https://ethereum-rpc.publicnode.com
+  explorerHostname: api.etherscan.io
+  explorerTokenEnv: ETHERSCAN_TOKEN
+  contracts:
+    l1NttManager:
+      name: NttManager
+      address: *l1NttManager
+      proxyName: ERC1967Proxy
+      implementation: *l1NttManagerImpl
+      proxyChecks: {}
+      checks:
+        NTT_MANAGER_VERSION: *NTT_MANAGER_VERSION # 1.1.0
+        chainId: *ETH_CHAIN_ID
+        getCurrentInboundCapacity:
+          - args: [ *ETH_CHAIN_ID ]
+            result: 0
+          - args: [ *BSC_CHAIN_ID ]
+            result: *RATE_LIMIT
+        getCurrentOutboundCapacity: *RATE_LIMIT # 30000000000000000000000
+        getInboundLimitParams:
+          - args: [ *ETH_CHAIN_ID ]
+            result: [0, 0, 0]
+          - args: [ *BSC_CHAIN_ID ]
+            result: [768000000000008, 768000000000008, 1722437087]
+        getInboundQueuedTransfer:
+          - args: [ *ZERO_BYTES32 ]
+            result: [ 0, 0, *ZERO_ADDRESS ]
+        getMigratesImmutables: false
+        getMode: *LOCKING_MODE
+        getOutboundLimitParams: [768000000000008, 767999999974408, 1722437087]
+        getOutboundQueuedTransfer:
+          - args: [ 0 ]
+            result: [ *ZERO_BYTES32, *ZERO_BYTES32, 0, 0, 0, *ZERO_ADDRESS, "0x" ]
+        getPeer:
+          - args: [ *ETH_CHAIN_ID ]
+            result: [ *ZERO_BYTES32, 0 ]
+          - args: [ *BSC_CHAIN_ID ]
+            result: [ *L2_NTT_MANAGER_BYTES32_ADDR, *DECIMALS ]
+        getThreshold: *THRESHOLD # 2
+        getTransceiverInfo: [[ true, true, 0 ],[ true, true, 1 ]] # registered, enabled, index
+        getTransceivers: [ *l1WormholeTransceiver, *l1AxelarTransceiver ] # 0xA1ACC1e6edaB281Febd91E3515093F1DE81F25c0,0x723AEAD29acee7E9281C32D11eA4ed0070c41B13
+        isMessageApproved:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isMessageExecuted:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isPaused: false
+        messageAttestations:
+         - args: [ *ZERO_BYTES32 ]
+           result: 0
+        mode: *LOCKING_MODE
+        nextMessageSequence: 38
+        owner: *L1_NEW_INTERIM_MSIG
+        pauser: *L1_NEW_INTERIM_MSIG
+        quoteDeliveryPrice:
+          - args: [ *BSC_CHAIN_ID, 0x ]
+            result: [[], 0]
```

Comment:
If `mustRevert: true` the field `result` is excess

Reply:
done